### DISCUSSION TITLE
github only displaying text and no images

### DISCUSSION THREAD

**Discussion by euijong** [community/community#21917] [New to GitHub]
<p>The Github page is only displaying text and no images (see screenshot below). I use Windows 10, and have tried multiple browsers (Chrome, Explorer, Firefox), but get the same result. Removing cache and any ad blockers does not solve the problem, so I am lost at this point.</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/1X/03827b897249fdb6c362468b8d7a4d5eed82b9d5.jpeg" data-download-href="/uploads/short-url/v2ZnDqc1vchLH696QEdCrDKJuZ.jpg?dl=1" title="github.JPG" rel="nofollow noopener"><img src="https://user-images.githubusercontent.com/16833512/181067521-dab62ffb-3d09-422c-a483-a18e0888d939.jpg" alt="github.JPG" data-base62-sha1="v2ZnDqc1vchLH696QEdCrDKJuZ" width="690" height="383" srcset="https://aws1.discourse-cdn.com/github/optimized/1X/03827b897249fdb6c362468b8d7a4d5eed82b9d5_2_690x383.jpg, https://aws1.discourse-cdn.com/github/optimized/1X/03827b897249fdb6c362468b8d7a4d5eed82b9d5_2_1035x574.jpg 1.5x, https://aws1.discourse-cdn.com/github/optimized/1X/03827b897249fdb6c362468b8d7a4d5eed82b9d5_2_1380x766.jpg 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/1X/03827b897249fdb6c362468b8d7a4d5eed82b9d5_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use xlink:href="#far-image"></use></svg><span class="filename">github.JPG</span><span class="informations">1582×879 79.3 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use xlink:href="#discourse-expand"></use></svg>
</div></a></div>

**Comment by mpboom** [comment, marked as answer]
<p>Did you try to use the Chrome DevTools Network tab to inspect network requests? Maybe you are behind some kind of firewall?</p>
